config/defaults.go

```diff
@@ -0,0 +1,308 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+)
+
+// DefaultConfigFilePath is in the root folder by default
+const DefaultConfigFilePath = "config.json"
+
+// General defaults
+const (
+	DebugDefault              = true
+	DebugProfilingDefault     = true
+	DisableITCDefault         = true
+	ImportBlockHeadersDefault = ""
+	RequestLoggingDefault     = true
+)
+
+// Authentication defaults
+const (
+	AuthAdminKeyDefault        = "xpub661MyMwAqRbcFaYeQLxmExXvTCjw9jjBRpifkoGggkAitXNNjva4TStLJuYjjEmU4AzXRPGwoECjXo3Rgqg8zQqW6UPVfkKtsrogGBw8xz7"
+	AuthRequireSigningDefault  = false
+	AuthSchemeDefault          = "xpub"
+	AuthSigningDisabledDefault = true
+)
+
+// Beef defaults
+const (
+	UseBeefDefault                  = true
+	PulseHeaderValidationURLDefault = "http://localhost:8000/api/v1/chain/merkleroot/verify"
+	PulseAuthTokenDefault           = "asd"
+)
+
+// Cachestore defaults
+const (
+	CacheEngineDefault = "freecache"
+)
+
+// Cluster defaults
+const (
+	ClusterCoordinatorDefault         = "redis"
+	ClusterPrefixDefault              = "bux_cluser_"
+	ClusterRedisURLDefault            = "localhost:6379"
+	ClusterRedisMaxIdleTimeoutDefault = "10s"
+	ClusterRedisUseTLSDefault         = false
+)
+
+// Datastore defaults
+const (
+	DatastoreAutoMigrateDefault = true
+	DatastoreDebugDefault       = false
+	DatastoreEngineDefault      = "sqlite"
+	DatastoreTablePrefixDefault = "xapi"
+)
+
+// MongoDB config keys
+const (
+	MongoDatabaseNameDefault = "xapi"
+	MongoTransactionsDefault = false
+	MongoURIDefault          = "mongodb://localhost:27017/xapi"
+)
+
+// SQL (MySQL, PostgreSQL) config keys
+const (
+	SQLDriverDefault                    = "postgresql"
+	SQLHostDefault                      = "localhost"
+	SQLNameDefault                      = "xapi"
+	SQLPasswordDefault                  = ""
+	SQLPortDefault                      = "5432"
+	SQLReplicaDefault                   = false
+	SQLSkipInitializeWithVersionDefault = true
+	SQLTimeZoneDefault                  = "UTC"
+	SQLTxTimeoutDefault                 = "10s"
+	SQLUserDefault                      = "postgres"
+)
+
+// SQLite config keys
+const (
+	SQLiteDatabasePathDefault = "./test-json.db"
+	SQLiteSharedDefault       = true
+)
+
+// Graphql defaults
+const (
+	GraphqlEnabledDefault        = true
+	GraphqlPlaygroundPathDefault = "/graphql"
+	GraphqlServerPathDefault     = "/graphql"
```

Comment:
Maybe we could consider removing the possibility of setting those GraphQL server and playground paths, and we could just put them statically in a code where it is used? This would mean that in default configuration we will end up with something like this:
```json
"graphql": {
   "enabled": false
}
```
Which is less eye caching and is easier to remove silently later 🙂